cdap-ui/app/cdap/api/dataprep.js

```diff
@@ -44,9 +44,9 @@ const MyDataPrepApi = {
   ping: apiCreator(dataSrc, 'GET', 'REQUEST', `${baseServicePath}/methods/usage`, { interval: 2000 }),
 
   // File System Browser
-  explorer: apiCreator(dataSrc, 'GET', 'REQUEST', `${baseServicePath}/methods/fs/explorer`),
-  readFile: apiCreator(dataSrc, 'GET', 'REQUEST', `${baseServicePath}/methods/fs/explorer/read`),
-  getSpecification: apiCreator(dataSrc, 'GET', 'REQUEST', `${baseServicePath}/methods/fs/explorer/specification`)
+  explorer: apiCreator(dataSrc, 'GET', 'REQUEST', `${baseServicePath}/methods/explorer/fs`),
```

Comment:
leftover from before when there was no mention of fs. Will be modified later on. This is to unblock people wanting to use the File browser